src/capitalsFirst.js

```diff
@@ -21,7 +21,12 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  const wordsArray = str.split(' ');
+  let upperWords = wordsArray.filter(word => word[0] >= 'A' && word[0] <= 'Z');
+  let lowerWords = wordsArray.filter(word => word[0] >= 'a' && word[0] <= 'z');
+  const resultArray = upperWords.concat(lowerWords);
```

Comment:
use spread operator instead of concat method 
concat has performance issues and looks ugly